.c8rc

```diff
@@ -0,0 +1,14 @@
+{
+  "all": true,
+  "exclude": [
+    "test/",
+    "coverage/",
+    "types/"
+  ],
+  "clean": true,
+  "check-coverage": true,
```

Comment:
Do we need to set this to true and have the rest of the settings? Doesn’t c8 by default checks all these?

Reply:
Setting this to `true` tells c8: "if below given threshold on any of the branches | lines | functions | statements" - exit process with code 1.
Setting to `false`, reports coverage anyway - but exit with 0.

Reply:
See here:
Where it's decided to checkCoverage based on given args or rc files:
https://github.com/bcoe/c8/blob/ff146b4dde004c62651b57c33cedd8353c94c423/lib/commands/report.js#L43

And here is exitCode 1 when per-file or global thresholds fail:
https://github.com/bcoe/c8/blob/ff146b4dde004c62651b57c33cedd8353c94c423/lib/commands/check-coverage.js#L59